policy/modules/kernel/filesystem.if

```diff
@@ -6700,6 +6700,7 @@ interface(`fs_read_efivarfs_files',`
         ')
 
         read_files_pattern($1, efivarfs_t, efivarfs_t)
+        read_files_pattern($1, init_var_run_t, init_var_run_t)
```

Comment:
This interface name is fs_read_efivarfs_files(), it should not contain other references.

Reply:
OK, should I add the pattern to read `init_var_run_t` in appropriate places instead?

Reply:
Interfaces are used when a type from another module is needed, supposedly `init_read_pid_files()`. I am also questioning the need as in the current policy version only 2 domains apart from systemd itself have read access to fhe files and a bunch to the directory:
```
# sesearch -A -t init_var_run_t -c file -p read |grep systemd
allow systemd_modules_load_t init_var_run_t:file { getattr ioctl lock open read };
allow systemd_tmpfiles_t non_auth_file_type:file { append create getattr ioctl link lock open read relabelfrom relabelto rename setattr unlink write };

# sesearch -A -t init_var_run_t -c dir |grep systemd
allow systemd_importd_t init_var_run_t:dir { add_name ioctl lock read remove_name write };
allow systemd_logind_t init_var_run_t:dir { add_name ioctl lock read remove_name write };
allow systemd_machined_t init_var_run_t:dir { add_name ioctl lock read remove_name write };
allow systemd_modules_load_t init_var_run_t:dir { ioctl lock read };
allow systemd_notify_t init_var_run_t:dir { add_name ioctl lock read remove_name write };
allow systemd_passwd_agent_t init_var_run_t:dir { add_name create ioctl lock read remove_name write };
allow systemd_resolved_t init_var_run_t:dir { add_name ioctl lock read remove_name write };
allow systemd_timedated_t init_var_run_t:dir { add_name ioctl lock read remove_name write };
allow systemd_tmpfiles_t non_auth_file_type:dir { add_name create getattr ioctl link lock open read relabelfrom relabelto remove_name rename reparent rmdir search setattr unlink write };
allow systemd_userdbd_t init_var_run_t:dir { add_name ioctl lock read remove_name write };
```
Also note some systemd domains use private type for files/dirs in the runtime directory.